extensions/opentelemetry/deployment/src/main/java/io/quarkus/opentelemetry/deployment/exporter/otlp/OtlpExporterProcessor.java

```diff
@@ -75,6 +77,13 @@ public boolean getAsBoolean() {
         }
     }
 
+    @BuildStep
+    void logging(BuildProducer<LogCategoryBuildItem> log) {
+        // Reduce the log level of the exporters because it's too much, and we do log important things ourselves.
+        log.produce(new LogCategoryBuildItem("io.opentelemetry.exporter.internal.grpc.GrpcExporter", Level.OFF));
+        log.produce(new LogCategoryBuildItem("io.opentelemetry.exporter.internal.http.HttpExporter", Level.OFF));
```

Comment:
Don't we override these?

Reply:
We use one here: https://github.com/quarkusio/quarkus/blob/95fea0b5da08c9ec968bfb8e3b42c621ce4412cb/extensions/opentelemetry/runtime/src/main/java/io/quarkus/opentelemetry/runtime/exporter/otlp/OTelExporterRecorder.java#L148

Reply:
Ah okay, thanks!

Reply:
We could override but if this (https://github.com/open-telemetry/opentelemetry-java/issues/6718) is not successful we might need to fork the entire artifact. I prefer to wait for that.

Reply:
Gotcha, now I remember